client/components/external-link/index.jsx

```diff
@@ -52,6 +52,10 @@ class ExternalLink extends Component {
 			}
 		);
 
+		if ( this.props.icon ) {
```

Comment:
Should we just default to `target="_blank"` for all `ExternalLinks` rather than tying it to the presence of the icon?

Reply:
I notice that the Gutenberg ExternalLink component does this:

https://github.com/WordPress/gutenberg/blob/01db6008fdf7b4c4e77647bb31dac8ca94a12704/packages/components/src/external-link/index.js#L28

I've done a sweep through all our uses of `ExternalLink` in Calypso and can't find a case where 'target="_blank"' wouldn't make sense (it's manually applied in nearly every case).

Reply:
Having `target="_blank"` as the default made the most sense to me too. Looking at the dev docs made me question that though. 

https://wpcalypso.wordpress.com/devdocs/design/external-link

If the icon is optional and we make `target="_blank"` the default, we'll run into the opposite issue where links are opening in new tabs without warning. 

What's the use case for using this component without the icon?

Reply:
> What's the use case for using this component without the icon?

One example is the full post title in Reader (the heading is linked to the external post):

<img width="1071" alt="Screen Shot 2019-06-19 at 13 34 47" src="https://user-images.githubusercontent.com/17325/59730589-08fc9a00-9297-11e9-89ad-774cd90bf292.png">

Reply:
> we'll run into the opposite issue where links are opening in new tabs without warning.

I agree. I think tying the behavior of opening in a new tab to the icon that indicates that being the case makes the most sense. I'm not sure if it's best approach to make this the default, however.

A link opening in the same tab is the default behavior of the browser. It is probably safe to assume that the majority of users are the most comfortable with that default behavior.

A link opening in a new tab has certain accessibility issues which can cause confusion for users relying on screen readers to navigate the website.

It is also worth noting that users can force a link to open in a new window/tab by [Meta Key]-clicking a link - so both behaviors are available to them when you don't force a new tab.

That's not to say that we would never want to have links open in a new tab but I don't feel that this should be the default behavior. And when we do set it, we would want to pair it with the icon to indicate a change in default behavior, as suggested in this PR.

Of course, this is just my 2¢. I'll leave it to y'all! :)